docs/source/example_notebooks/figsizes.ipynb

```diff
@@ -190,25 +207,86 @@
+      "text/plain": [
+       "<Figure size 234x144.62 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams.update(figsizes.aistats2022_half())\n",
+    "fig, ax = plt.subplots()\n",
+    "ax.plot([1.0, 2.0], [3.0, 4.0])\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da7aa338",
+   "metadata": {},
+   "source": [
+    "### Figure sizes that match your subplot layouts\n",
+    "When working with `plt.subplots`, provide the `nrows` and `ncols` also to the figsize functions to get consistent subplot sizes."
```

Comment:
How do you feel about adding some more details here? I.e. what I told you offline. I think it would help a user to get a glimpse into the `why` at this point. 
You are very much invited to use your own words, but here is another explanation that might serve as a base:

The `nrows` and `ncols` argument essentially dictates the overall figure _height_. Why? Because each _subplot_ is fixed to a specific format (usually the golden ratio), and the figure width is commonly tied to the specific journal style. The remaining degree of freedom, the overall figure height, is adapted to make things look clean.

Reply:
Good suggestion, and thanks for the proposal! I just took that one quite directly :)